### PR TITLE
Add Iranian messengers

### PR DESCRIPTION
**Pull request by xmha97**
Dear developer(s)
"‌Bale" and "Eitaa" messengers are widely used in Iran and I want you to add them to this app
Thank you for making this app

### CODE REVIEW

File: recipes/bale/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "bale",
+  "name": "Bale",
+  "version": "1.0.1",
```
Comment (vraravam) [inline recipes/bale/package.json:4]: pls change this to `1.0.0`

File: recipes/eitaa/package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "Eitaa",
+  "name": "Eitaa",
+  "version": "1.0.1",
```
Comment (vraravam) [inline recipes/eitaa/package.json:4]: pls change this to `1.0.0`

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by xmha97** [comment]
@vraravam Hi, I fix the version.

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
@xmha97 - please see the error shown by the CI run and fix it. Thanks for your contribution!

**Comment by xmha97** [comment]
> @xmha97 - please see the error shown by the CI run and fix it. Thanks for your contribution!

@vraravam - I think the error is fixed now.

**Comment by vraravam** [comment]
Thanks for your contribution @xmha97 

@all-contributors please add @xmha97 for code

**Pull request merged**